### PR TITLE
Enable setting author via uw group

### PR DESCRIPTION
**Pull request by fanglinfang**

### CODE REVIEW

File: cron [hunk 1]
```diff
@@ -1,7 +1,8 @@
 MAILTO=fanglin@uw.edu
```
Comment (ezturner) [inline cron]: Could this be converted to a mailman list?

Comment (fanglinfang) [reply cron]: for now just me

File: sis_provisioner/account_managers/author_loader.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                self.remove_author_role(bri_acc)
+            else:
+                existing_bri_authors.add(uwnetid)
+
+        # 2. add new authors
+        for netid in list(self.cur_author_set):
+            if self.in_uw_groups(netid) is False:
+                continue
+            if netid not in existing_bri_authors:
+                self.add_author_role(netid)
```
Comment (ezturner) [inline sis_provisioner/account_managers/author_loader.py]: Have you thought about using list addition and subtraction to get the diff here?

Comment (fanglinfang) [reply sis_provisioner/account_managers/author_loader.py]: Ah,  thank you!

**Pull request merged**